services/src/test/scala/cromwell/services/instrumentation/impl/stackdriver/StackdriverConfigSpec.scala

```diff
@@ -0,0 +1,99 @@
+package cromwell.services.instrumentation.impl.stackdriver
+
+import com.typesafe.config.ConfigFactory
+import cromwell.core.TestKitSuite
+import org.scalatest.{BeforeAndAfterAll, FlatSpecLike, Matchers}
+
+import scala.concurrent.duration._
+
+class StackdriverConfigSpec extends TestKitSuite with FlatSpecLike with BeforeAndAfterAll with Matchers {
+  behavior of "StackdriverConfig"
+
+  val googleConfig = ConfigFactory.parseString(
+    s"""
+       |google {
+       |  application-name = "cromwell"
+       |  auths = [
+       |    {
+       |      name = "application-default"
+       |      scheme = "application_default"
+       |    }
+       |  ]
+       |}
+      """.stripMargin
+  )
+
+  it should "correctly parse all config" in {
+    val config = ConfigFactory.parseString(
+      """
+        |auth = "application-default"
+        |google-project = "my-project"
+        |flush-rate = 1 minute
+        |cromwell-instance-identifier = "cromwell-101"
+        |cromwell-instance-role = "backend"
+        |cromwell-perf-test-case = "perf-test-1"
+      """.stripMargin
+    )
+
+    val stackdriverConfig = StackdriverConfig(config, googleConfig)
+
+    stackdriverConfig.auth.name shouldBe "application-default"
+    stackdriverConfig.googleProject shouldBe "my-project"
+    stackdriverConfig.flushRate shouldBe 1.minute
+    stackdriverConfig.cromwellInstanceIdentifier shouldBe Some("cromwell-101")
+    stackdriverConfig.cromwellInstanceRole shouldBe Some("backend")
+    stackdriverConfig.cromwellPerfTestCase shouldBe Some("perf-test-1")
```

Comment:
`Some.apply()` when used incorrectly causes very strange, very hard to debug problems. As `Option.apply()` can do the same exact thing as `Some.apply()` with a tiny performance penalty, the latter is considered banned from the codebase.

TL;DR Use `Option()` aka `Option.apply()`.

Reply:
Will change it to `Option(...)`. Thanks!